edit/edit.js

```diff
@@ -1334,7 +1364,20 @@ function setStyleMeta(style) {
   $('#url').href = style.url || '';
 }
 
-function initWithStyle({style, codeIsUpdated}) {
+function initWithStyle({style}) {
+  // FIXME: what does codeIsUpdated do?
+  if (!style.usercss) {
+    return _initWithStyle({style});
+  }
+
+  if (editor) {
+    editor.replaceStyle(style);
+  } else {
+    editor = createSourceEditor(style);
+  }
+}
+
+function _initWithStyle({style, codeIsUpdated}) {
```

Comment:
This is a meaningless underscore, it doesn't tell anything. Either rename the function or rework the code to remove the need to use this function.